golang/iotexapi/read_state.pb.go

```diff
@@ -9,8 +9,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.21.12
```

Comment:
let's stick to the current version, otherwise it brings in lots of unnecessary changes